source/administration/update_mattermost.sh

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env bash
```

Comment:
If this is a new doc it should be in the table of contents: https://docs.mattermost.com/guides/administrator.html.

Also, I'm not able to get a proper preview of this page when clicking "View file".

Reply:
The doc is already in the table of content.

The file that you are talking about (which is not proper previewed) is not a documentation text (a restructured fie) but rather a Bash script. This is code, this is thus normal you don't see any proper preview.